selfservice/hook/session_destroyer_test.go

```diff
@@ -0,0 +1,43 @@
+package hook_test
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ory/viper"
+
+	"github.com/ory/kratos/driver/configuration"
+	"github.com/ory/kratos/identity"
+	"github.com/ory/kratos/internal"
+	"github.com/ory/kratos/selfservice/hook"
+	"github.com/ory/kratos/session"
+	"github.com/ory/kratos/x"
+)
+
+func TestSessionIssuer(t *testing.T) {
+	_, reg := internal.NewRegistryDefault(t)
+	viper.Set(configuration.ViperKeyURLsSelfPublic, "http://localhost/")
+	viper.Set(configuration.ViperKeyDefaultIdentityTraitsSchemaURL, "file://./stub/stub.schema.json")
+
+	var r http.Request
+	h := hook.NewSessionIssuer(reg)
+
+	t.Run("method=sign-in", func(t *testing.T) {
+		w := httptest.NewRecorder()
+		sid := x.NewUUID()
+
+		i := identity.NewIdentity("")
+		require.NoError(t, reg.IdentityPool().CreateIdentity(context.Background(), i))
+		require.NoError(t, h.ExecuteLoginPostHook(w, &r, nil, &session.Session{ID: sid, Identity: i}))
```

Comment:
This test isn't actually testing the functionality of what you've implemented. Instead, you need to do something along the lines of:

1. Create a new identity
2. Create 2 or more sessions for that identity
3. Run `ExecuteLoginPostHook`
4. Check if these 2 (or more) sessions have been removed from the store. To prove that they're gone, `GetSession` should return an error of type `sqlcon.ErrNoRows`.